src/cli/moz-cache.js

```diff
@@ -1,5 +1,6 @@
 import { Scheduler } from 'taskcluster-client';
 import slugid from 'slugid';
+import createHash from '../hash';
```

Comment:
I was just using it to be consistent with what we were doing elsewhere.  I'm not sure the reasoning of moving this into it's own module but I would assume so that wherever we need a hash for an indexed namespace, it's always consistently using the same algorithm as well as always returning a hex string.